test/jdk/java/foreign/TestFunctionDescriptor.java

```diff
@@ -141,4 +141,29 @@ public void testIllegalInsertArgOutOfBounds() {
         fd.insertArgumentLayouts(2, C_INT);
     }
 
+    @Test(expectedExceptions = IllegalArgumentException.class)
+    public void testBadPaddingInVoidFunction() {
+        FunctionDescriptor.ofVoid(MemoryLayout.paddingLayout(8));
+    }
+
+    @Test(expectedExceptions = IllegalArgumentException.class)
+    public void testBadPaddingInNonVoidFunction() {
+        FunctionDescriptor.ofVoid(MemoryLayout.paddingLayout(8));
+    }
```

Comment:
This test is the same as the one above. Looks like you meant to call `of` here? (instead of `ofVoid`)